auth/src/main/java/info/rmapproject/auth/dao/ApiKeyDaoImpl.java

```diff
@@ -114,7 +114,7 @@ public ApiKey getApiKeyByKeyUri(String keyUri) throws RMapAuthException {
         query.setParameter("keyUri",keyUri);
 		List<ApiKey> apiKeys = query.list();
 		if (apiKeys != null && !apiKeys.isEmpty()) {
-	        logger.info("Api key list loaded successfully");
+	        LOG.info("Api key list loaded successfully");
```

Comment:
Do you think it would be worth-while to review the logging levels prior to a "production" release of each log message and see what is truly INFO vs DEBUG vs WARN.  Some of that may be hard to intuit, of course, and may rely on being inside the head of a sysadmin as they review system logs when there's a problem.  I just wonder about normal INFO-level logs being noisy, without providing much value to a sysadmin.

Reply:
Issue #59 was a vague catch-all, but we may not have time to fully address before production.  There isn't much logging, so reviewing what's there should be easy enough, not great though.